packages/@o3r/components/builders/component-extractor/helpers/component/component-config.extractor.ts

```diff
@@ -365,10 +365,36 @@ export class ComponentConfigExtractor {
           if (ts.isVariableDeclaration(declarationNode)) {
             let isConfigImplementation = false;
             declarationNode.forEachChild((vNode) => {
-              if (ts.isTypeReferenceNode(vNode) && configurationInformationWrapper.configurationInformation!.name === vNode.getText(this.source)) {
+              if (
```

Comment:
Could not manage to understand from the changes if this is a backward compatible update. So, I am asking it ? :)

Reply:
Yes it supposed to be